src/components/header/OverflowEdgeIndicator.js

```diff
@@ -53,9 +53,11 @@ class OverflowEdgeIndicator extends PureComponent {
   }
 
   _updateIndicatorStatus() {
-    if (this._container && this._contentsWrapper) {
-      const containerRect = this._container.getBoundingClientRect();
-      const contentsRect = this._contentsWrapper.getBoundingClientRect();
+    const container = this._container;
+    const contentsWrapper = this._contentsWrapper;
```

Comment:
same thing than above: we need to assign these properties to constants so that Flow is sure we don't nullify them.

Reply:
Flow didn't care before because we improperly typed these properties.